polymode-core.el

```diff
@@ -953,7 +953,8 @@ Parents' hooks are run first."
     text-scale-mode-amount
     truncate-lines
     truncate-partial-width-windows
-    word-wrap)
+    word-wrap
+    mc--this-command)
```

Comment:
Could you please leave a brief comment explaining why this internal variable needs to be moved. Just to be helpful to whoever needs to deal with `mc` integration later.